cli/src/cmd/exec.rs

```diff
@@ -501,6 +554,37 @@ impl<'a> From<(&'a str, &'a clap::ArgMatches)> for Subcommand<'a> {
                     with_context,
                 )
             }
+            ("plugin", args) => match args.subcommand() {
+                Some(("call", args)) => {
+                    let identifier = args
+                        .get_one::<String>("identifier")
```

Comment:
consider `id`, but whatever is used in the installation cmd should be consistent